### PR TITLE
Euwe-1-Beta1 announcement

### PR DESCRIPTION
**Pull request by JPrause**

### CODE REVIEW

File: data/deploy_releases.yml [hunk 1]
```diff
@@ -1,9 +1,9 @@
-stable:
-  name: Stable (darga-4.1)
-  filename: darga-4.1
-# prerelease:
-#   name: Prerelease (darga-1-rc1)
-#   filename: darga-1-rc1.1
+# stable:
+#  name: Stable (darga-4.1)
+#  filename: darga-4.1
```
Comment (simaishi) [inline data/deploy_releases.yml]: The 'stable' section will need to stay (not commented out)

File: source/blog/2016-10-07-announcing-euwe-1-beta1.html.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+comments: true
+published: true
+---
+
+On behalf of the ManageIQ team, we're pleased to announce the Beta release for ManageIQ Euwe! 
+
+Euwe is the next milestone release for ManageIQ cloud and virtualization management platform. The update is packed with new features and improvements. Highlights include:
+
+* Cinder Storage Manager
+* Swift Storaeg Manager
```
Comment (Fryguy) [inline source/blog/2016-10-07-announcing-euwe-1-beta1.html.md]: typo `Storage`

**Comment by mfeifer** [comment]
Suggested focus points:

* Centralized Administration
* Performance Improvements (including SmartState)
* Providers: Core
* PostgreSQL HA

I know there's more...see the changelog.

**Comment by mfeifer** [comment]
Intro paragraph

Euwe is the next milestone release for ManageIQ cloud and virtualization management platform.  With each release Manageiq gets more robust  and feature complete across providers.  In this release, we not only attended to providers and chargeback, but we also made some essential improvements to the appliance itself including performance, and high availability.

**Comment by mfeifer** [comment]
At the end:

If you want to hear about features, improvements, and fixes directly check out the [Last Week in ManageIQ blot posts](http://manageiq.org/blog/tags/LWIMIQ/).

File: source/blog/2016-10-07-announcing-euwe-1-beta1.html.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+comments: true
+published: true
+tags: releases announcements
+---
+
+On behalf of the ManageIQ team, we're pleased to announce the Beta release for ManageIQ Euwe! 
+
+Euwe is the next milestone release for ManageIQ cloud and virtualization management platform. The update is packed with new features and improvements. Highlights include:
+
+* Centralized Administration
```
Comment (mfeifer) [inline source/blog/2016-10-07-announcing-euwe-1-beta1.html.md]: * Darga timelines have been replaced by PatternFly timelines in Euwe.  

* Notification drawer has been added using ActionCable with PatternFly's notification drawer powered by Angular.

**Pull request closed without merging**